sass/main.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../../../template';
+
+
```

Comment:
I think i might be doing bad things here.... let me change this.
added template to scss and this PR